smart_control/reinforcement_learning/utils/config.py

```diff
@@ -35,86 +34,142 @@
 # Path to the root directory of the project:
 ROOT_DIR = os.path.join(os.path.dirname(__file__), "..", "..", "..")
 
-DATA_PATH = os.path.join(ROOT_DIR, "smart_control", "configs", "resources", "sb1")
-CONFIG_PATH = os.path.join(ROOT_DIR, "smart_control", "configs", "resources", "sb1", "train_sim_configs")
-METRICS_PATH = os.path.join(ROOT_DIR, "smart_control", "reinforcement_learning", "experiment_results", "metrics")
-RENDERS_PATH = os.path.join(ROOT_DIR, "smart_control", "reinforcement_learning", "experiment_results", "renders")
-OUTPUT_DATA_PATH = os.path.join(ROOT_DIR, "smart_control", "reinforcement_learning", "data", "starter_buffers")
-EXPERIMENT_RESULTS_PATH = os.path.join(ROOT_DIR, "smart_control", "reinforcement_learning", "experiment_results")
+DATA_PATH = os.path.join(
```

Comment:
let's revert and ignore this change and leave these long lines so the filepaths are easier to read and compare with each other